phenex/phenotypes/continuous_coverage_phenotype.py

```diff
@@ -15,93 +19,112 @@
 
 class ContinuousCoveragePhenotype(Phenotype):
     """
-    A phenotype based on continuous coverage within an observation period.
-
-    This class helps generate SQL queries to filter a population based on
-    continuous coverage criteria within the observation period.
-
-    :param domain: The domain of the phenotype, default is 'observation_period'. The domain
-        key is used at runtime to determine which table to run on.
-    :param coverage_period_min: The minimum coverage period for the phenotype with a default
-        of 0 days. The operator must be '>=' or '>'.
-    :param return_date: An optional return date for the phenotype result. Possible values are
-        "first" and "last", where "first" is the beginning of the coverage period containing
-        the index date and "last" in the end of the coverage period containing the index date.
-
-    Example usage: Find all patients with at least 90 days of continuous coverage
-    --------------
-    >>> coverage_min_filter = ValueFilter(">=", 90)
-    >>> phenotype = ContinuousCoveragePhenotype(coverage_period_min=coverage_min_filter)
+    ContinuousCoveragePhenotype identifies patients based on duration of observation data. ContinuousCoveragePhenotype requires an anchor phenotype, typically the entry criterion. It then identifies an observation time period that contains the anchor phenotype. The phenotype can then be used to identify patients with a user specified continuous coverage before or after the anchor phenotype.
+
+    There are two primary use cases for ContinuousCoveragePhenotype:
+        1. Identify patients with some minimum duration of coverage prior to anchor_phenotype date e.g. "identify patients with 1 year of continuous coverage prior to index date"
+        2. Determine the date of loss to followup (right censoring) i.e. the duration of coverage after the anchor_phenotype event
+
+    ## Data for ContinuousCoveragePhenotype
+    This phenotype requires a table with PersonID and a coverage start date and end date. Depending on the datasource used, this information is a separate ObservationPeriod table or found in the PersonTable. Use an ObservationPeriodTableMapper to map required coverage start and end date columns.
+
+    | PersonID    |   coverageStartDate  |   coverageEndDate  |
```

Comment:
I suspect this won't render correctly. Have you checked? For me, I needed to put ```markdown in front to get tables to render